Doppler.Currency.Test/Integration/CurrencyControllerTests.cs

```diff
@@ -22,11 +23,14 @@ public CurrencyControllerTests(TestServerFixture testServerFixture)
         }
 
         [Theory]
-        [InlineData("1-2-2012", "ARS")]
-        [InlineData("01-02-2012", "mxn")]
-        [InlineData("01-2-2012", "MXN")]
-        [InlineData("1-02-2012", "mXn")]
-        public async Task GetCurrency_ShouldBeHttpStatusCodeOk_WhenDateAndCurrencyCodeAreCorrectly(string dateTime, string currencyCode)
+        [InlineData("1-2-2012", "ARS", "Peso Argentino")]
+        [InlineData("01-02-2012", "mxn", "Peso Mexicano")]
+        [InlineData("01-2-2012", "MXN", "Peso Mexicano")]
+        [InlineData("1-02-2012", "mXn", "Peso Mexicano")]
+        public async Task GetCurrency_ShouldBeHttpStatusCodeOk_WhenDateAndCurrencyCodeAreCorrectly(
+            string dateTime, 
+            string currencyCode,
+            string currencyName)
```

Comment:
In my opinion, the resulting country code should always have the same uppercase text format.

```suggestion
        [InlineData("1-02-2012", "mXn", "Peso Mexicano", "MXN")]
        public async Task GetCurrency_ShouldBeHttpStatusCodeOk_WhenDateAndCurrencyCodeAreCorrectly(
            string dateTime, 
            string requestCurrencyCode,
            string currencyName,
            string expectedCurrencyCode)
```

Reply:
Done I updated the code in this commit https://github.com/FromDoppler/doppler-currency/pull/11/commits/007396deff350dd66b701ad904900510c225c0e2

Reply:
👍